src/TaCo/connectionBuilder.js

```diff
@@ -0,0 +1,43 @@
+(function dsbuilder(attr){
+    var params = {};
+
+    // Set host information in connection string
+    params["SERVER"] = attr[connectionHelper.attributeServer];
+    params["PORT"] = attr[connectionHelper.attributePort];
+
+    // Set authentication values in connection string
+    var authAttrValue = attr[connectionHelper.attributeAuthentication];
+    if (authAttrValue == "auth-none"){
+        params["Auth"] = "NONE";
+    } else if (authAttrValue == "AWS_SIGV4"){
+        params["Auth"] = "AWS_SIGV4";
+        params["Region"] = attr[connectionHelper.attributeVendor1];
+    } else if (authAttrValue == "BASIC"){
+        params["Auth"] = "BASIC";
+        params["UID"] = attr[connectionHelper.attributeUsername];
+        params["PWD"] = attr[connectionHelper.attributePassword];
+    }
+
+    // Set SSL value in connection string 
+    if (attr[connectionHelper.attributeSSLMode] == "require"){
+        params["useSSL"] = "1";
+    }
```

Comment:
Let's explicitly set this to `"0"` in an `else` case

Reply:
done